src/app/examples/segment-example/segment-example.component.tns.html

```diff
@@ -0,0 +1,7 @@
+<StackLayout class="segment-example">
+        <kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelected($event)" [selectedItem]="selectedItem">
```

Comment:
Follow [Angular Style Guide 05-16](https://angular.io/guide/styleguide#style-05-16): "Do name event handler methods with the prefix on followed by the event name."
```suggestion
        <kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelect($event)" [selectedItem]="selectedItem">
```